Buildings/Experimental/DHC/EnergyTransferStations/Cooling/DirectUncontrolled.mo

```diff
@@ -0,0 +1,175 @@
+  without in-building pumping or deltaT control"
+  extends
+    Buildings.Experimental.DHC.EnergyTransferStations.BaseClasses.PartialETS(
+    final typ=DHC.Types.DistrictSystemType.Cooling,
+    final have_weaBus=false,
+    final have_chiWat=true,
+    final have_heaWat=false,
+    final have_hotWat=false,
+    final have_eleHea=false,
+    final nFue=0,
+    final have_eleCoo=false,
+    final have_pum=false,
+    final have_fan=false,
+    nPorts_aChiWat=1,
+    nPorts_bChiWat=1);
+  parameter Modelica.Units.SI.MassFlowRate m_flow_nominal(
+    final min=0,
+    final start=0.5)
+    "Nominal mass flow rate"
+    annotation(Dialog(group="Nominal condition"));
+  // Pressure drops
+  parameter Modelica.Units.SI.PressureDifference dpSup(
+    final min=0,
+    displayUnit="Pa")=5000
+    "Pressure drop in the ETS supply side";
+  parameter Modelica.Units.SI.PressureDifference dpRet(
+    final min=0,
+    displayUnit="Pa")=5000
+    "Pressure drop in the ETS return side";
+  Buildings.Fluid.FixedResistances.PressureDrop pipSup(
+    redeclare final package Medium=MediumSer,
+    final allowFlowReversal=false,
+    final m_flow_nominal=m_flow_nominal,
+    final dp_nominal=dpSup)
+    "Supply pipe"
+    annotation (Placement(transformation(extent={{20,-290},{40,-270}})));
+  Buildings.Fluid.FixedResistances.PressureDrop pipRet(
+    redeclare final package Medium=MediumSer,
+    final allowFlowReversal=false,
+    final m_flow_nominal=m_flow_nominal,
+    final dp_nominal=dpRet)
+    "Return pipe"
+    annotation (Placement(transformation(extent={{20,190},{40,210}})));
+  Buildings.Fluid.Sensors.TemperatureTwoPort senTDisSup(
+    redeclare final package Medium=MediumSer,
+    final m_flow_nominal=m_flow_nominal)
+    "District supply temperature sensor"
+    annotation (Placement(transformation(extent={{-140,-290},{-120,-270}})));
+  Buildings.Fluid.Sensors.TemperatureTwoPort senTBuiRet(
+    redeclare final package Medium=MediumSer,
+    final m_flow_nominal=m_flow_nominal)
+    "Building return temperature sensor"
+    annotation (Placement(transformation(extent={{-178,210},{-158,190}})));
+  Buildings.Fluid.Sensors.TemperatureTwoPort senTBuiSup(
+    redeclare final package Medium = MediumSer,
+    final m_flow_nominal=m_flow_nominal)
+    "Building supply temperature sensor"
+    annotation (Placement(transformation(extent={{240,190},{260,210}})));
+  Buildings.Fluid.Sensors.TemperatureTwoPort senTDisRet(
+    redeclare final package Medium = MediumSer,
+    final m_flow_nominal=m_flow_nominal)
+    "District return temperature sensor"
+    annotation (Placement(transformation(extent={{240,-270},{260,-290}})));
+  Buildings.Fluid.Sensors.MassFlowRate senMasFlo(
+    redeclare final package Medium=MediumSer)
+    "District supply mass flow rate sensor"
+    annotation (Placement(transformation(extent={{-78,-290},{-58,-270}})));
+  Modelica.Blocks.Math.Add dTdis(
+  final k1=-1,
```

Comment:
add one Tab to move in a little bit

Reply:
Thank you for the review, have already revised it accordingly.